infra/nnfw/python/setup.py

```diff
@@ -0,0 +1,58 @@
+from setuptools import setup, find_packages
+import os
+import shutil
+
+architecture_directory = ['x86_64', 'armv7l', 'aarch64']
+package_directory = 'nnfwapi'
+packaging_directory = ['build', 'dist', package_directory + '.egg-info']
+so_list = []
+
+try:
+    # remove packaging directory
+    for packaging_dir in packaging_directory:
+        if os.path.exists(packaging_dir):
+            print(f"Deleting existing directory '{packaging_dir}'...")
+            shutil.rmtree(packaging_dir)
+
+    # remove architectory directory
+    if os.path.exists(package_directory):
+        for arch_dir in architecture_directory:
+            arch_path = os.path.join(package_directory, arch_dir)
+            if os.path.exists(arch_path):
+                print(f"Deleting existing directory '{arch_path}'...")
+                shutil.rmtree(arch_path)
+
+        # make architecture_directory and copy .so files to the directories
+        for arch_dir in architecture_directory:
+            arch_path = os.path.join(package_directory, arch_dir)
+            os.makedirs(arch_path)
+            print(f"Created directory '{arch_path}'")
+
+            so_dir = os.path.join('../../../Product', arch_dir + '-linux.release/out/lib')
```

Comment:
@hseok-oh 
Is there a good way to avoid using relative path?

Reply:
I don't good at python packaging. I just checked python code itself.

Reply:
It would be good to remove relative path. I guess this code may not work if `setup.py` is executed out of the same directory.

If so, what about using current script directory as starting point? I searched and 
 `os.path.dirname(os.path.abspath(__file__))` returns the directory.